Frontend/ui-driver/src/Screens/Types.purs

```diff
@@ -446,17 +447,33 @@ type DriverDetailsScreenState = {
   props :: DriverDetailsScreenStateProps
 }
 
+data KeyboardModalType = ALTERNATE_NUMBER | OTP | NONE
```

Comment:
instead oh `ALTERNATE_NUMBER` make it `MOBILE_NUMBER` which makes it more generic.